Automattic/jetpack
# Remove Pocket Casts embed block variation

#### Changes proposed in this Pull Request:
This PR removes the Pocket Casts embed block variation, it was added on https://github.com/Automattic/jetpack/pull/23740 for internal dev purposes.

The Pocket Casts block will be moved to Gutenberg on https://github.com/WordPress/gutenberg/pull/41006.

#### Jetpack product discussion
pdDOJh-bf-p2 and pdeCcb-dg-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Open the block editor
2. Search for the Pocket Casts block
3. The block should not appear in the results
<img width="367" alt="Screen Shot 2022-05-19 at 15 33 58" src="https://user-images.githubusercontent.com/3113712/169376570-1614cde7-6886-4d4d-8562-80f2d6e65976.png">
